devtools-3d/README.md

```diff
@@ -4,16 +4,20 @@
 
 This demo page is useful to show some of the most useful features of the 3D View tool in Microsoft Edge DevTools.
 
-* Read the blog post about the tool here (link coming soon).
-* Watch the video about the tool here (link coming soon).
-* Read documentation about the tool [here](https://learn.microsoft.com/microsoft-edge/devtools/3d-view/).
+* Read the blog post about the tool here (link coming soon).<!-- todo -->
```

Comment:
blog post & video about 3D tool available?
@captainbrosset

Reply:
Blog: https://blogs.windows.com/msedgedev/2022/06/21/debug-the-web-in-3d-with-the-3d-view-tool/
Video: https://www.youtube.com/watch?v=LVb1Xzzd72w

Reply:
See also:
* [Navigate webpage layers, z-index, and DOM using the 3D View tool](https://learn.microsoft.com/microsoft-edge/devtools/3d-view/) - Documentation.
* [Debug the web in 3D with the 3D View tool](https://blogs.windows.com/msedgedev/2022/06/21/debug-the-web-in-3d-with-the-3d-view-tool/) - Blog post.
* [Debug the web in 3D](https://www.youtube.com/watch?v=LVb1Xzzd72w) - Video.